_adr/88/index.adoc

```diff
@@ -1,48 +1,70 @@
 ---
 num: 88
-title: Authorization for control plane admin APIs
-status: "Draft" # One of Draft, Accepted, Rejected
+title: Authorization for fleet-manager admin APIs
+status: "Draft"
 authors:
-  - "akoserwal"
+  - "ppaszki"
 tags:
-  - "sso" # e.g. kafka, connectors, registry
+  - "sso"
 applies_padrs: # What PADRs does this ADR apply?
 applies_patterns: # What APs does this ADR apply?
 ---
 
-// Top style tips:
-// * Use one sentence per line
-// * No unexpanded acronyms
-// * No undefined jargon
-
-// No need for a title heading, it's added by the template
+:rover-groups: https://rover.redhat.com/groups
 
 ## Context and Problem Statement
-// What is the background against which this decision is being taken?
+The purpose of this ADR is to define a standardized approach for securing admin API in a fleet manager
 
 ## Goals
-// Bulleted list of outcomes that this ADR, if accepted, should help achieve
+* Provide standardized way to secure admin API endpoints for fleet managers
 
 ## Non-goals
-// Bulleted list of outcomes that this ADR is not trying to achieve.
 
 ## Current situation
```

Comment:
Please can you add about the `Current situation.

Reply:
but for generic ADR there isn't really current situation, am I right?

Reply:
yes, I agree.

Reply:
so anything else to be added here or in this ADR? Can I remove the `draft` from the PR description?

Reply:
yes, looks good to me. You can remove the draft remark

Reply:
Better, I think, to say what you've said here: "This is an unsolved problem in the general case, so there is no current situation".